src/BlockTemplatesController.php

```diff
@@ -103,4 +104,37 @@ public function theme_has_template( $template_name ) {
 		return is_readable( get_template_directory() . '/block-templates/' . $template_name . '.html' ) ||
 			is_readable( get_stylesheet_directory() . '/block-templates/' . $template_name . '.html' );
 	}
+
+	/**
+	 * Checks whether a block template with that name exists in Woo Blocks
+	 *
+	 * @param string $template_name Template to check.
+	 * @return boolean
+	 */
+	public function default_block_template_is_available( $template_name ) {
+		if ( ! $template_name ) {
+			return false;
+		}
+
+		return is_readable(
+			$this->templates_directory . '/' . $template_name . '.html'
+		);
+	}
+
+	/**
+	 * Renders the default block template from Woo Blocks if no theme templates exist.
+	 */
+	public function render_block_template() {
+		if ( is_embed() || function_exists( 'gutenberg_supports_block_templates' ) && ! gutenberg_supports_block_templates() ) {
```

Comment:
Because we're returning early, this will allow execution of the logic below the condition on sites that don't have Gutenberg activated. We should exclude that case as well.
```suggestion
		if ( is_embed() || ! function_exists( 'gutenberg_supports_block_templates' ) || ! gutenberg_supports_block_templates() ) {
```